website/search/elastic_search.py

```diff
@@ -440,6 +441,9 @@ def search_contributor(query, page=0, size=10, exclude=None, current_user=None):
         most recent employment and education, gravatar URL of an OSF user
 
     """
+    page = int(request.args.get('page', 0))
```

Comment:
What if the client passes an invalid value, e.g. a string that isn't an integer? Handle errors correctly here.